Platform/CMakeLists.txt

```diff
@@ -19,6 +19,43 @@ if(WIN32 AND NOT CMAKE_HOST_SYSTEM_NAME MATCHES Windows)
  return()
 endif()
 
+if(MINGW)
+    if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+        set(ARCH64_BITS TRUE)
+    else()
+        set(ARCH64_BITS FALSE)
```

Comment:
You can use the `PLATFORM_ABI` variable that is set in the root CMakeLists.txt. https://github.com/Gjacquenot/simbody/blob/master/CMakeLists.txt#L198